src/makeCalculator.js

```diff
@@ -4,7 +4,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {};
+
+  calculator.result = 0;
+
+  calculator.operate = (callback, number) => {
+    calculator.result = callback(number);
+
+    return calculator;
+  };
```

Comment:
Bug: The `operate` method should not replace the `result` with the return value of the callback. It should instead add, subtract, multiply or divide the `result` by the number passed, depending on the callback. Currently, your `operate` method simply replaces the `result` with the return value of the callback, which is not correct. Instead, you should use the callback to modify the existing `result` value. For example, if the callback is `add`, you should add the number to the `result`, not replace the `result` with the number.